app/posts/apply-for-teacher-training/2022-05-13-degree-autocomplete-changes.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Degree autocomplete changes
+description: Why we made changes to the autocomplete
+date: 2022-05-13
+---
+
+![TODO](degree-autocomplete-changes.png "Old and new autocompletes")
+
+The section of our application where candidates enter their degree information had ‘autocomplete’ fields for degree type (BA, BSc, etc), university name and degree subject.
+
+These fields allowed users to either pick from one of the options after typing in 2 or more characters, or enter their own answer.
+
+The options provided were taken from the [HESA ITT data collection guidelines](https://www.hesa.ac.uk/collection/c21053/index).
+
+From looking at the data, we noted that a high proportion of candidates were not selecting options from the autocomplete.
+
+This led to messy data which made analysis harder, for example by having lots of different spellings and abbreviations for the same universities or subjects.
+
+We were also concerned that for the subject field, candidate may have not always realised that they could type in a subject not in the list, and may have picked the closest match instead. This may be frustrating, or cause minor issues later in the process when they have to produce evidence of their degree qualification.
```

Comment:
```suggestion
We were also concerned that for the subject field, the candidate may have not always realised that they could type in a subject not in the list, and may have picked the closest match instead. This may be frustrating, or cause minor issues later in the process when they have to produce evidence of their degree qualification.
```